### PR TITLE
raidboss: improve zenos protean text

### PR DESCRIPTION
**Pull request by jacob-keller**
- raidboss: trigger on gain doom, not losing doom
- raidboss: improve Zenos "protean" tell

### CODE REVIEW

File: ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ const triggerSet: TriggerSet<Data> = {
       alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Protean',
+          en: 'Protean (Spread Out)',
```
Comment (quisquous) [inline ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts:99]: Protean means something different to me than spread.  I understand that not everybody knows what this means, but I think it's better to call things consistently across multiple fights.  Folks can change it in the cactbot ui if they don't like it.

Comment (Jessidhia) [reply ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts:99]: “Protean”, however, is meaningless to people who didn’t raid Alex…

**Comment by jacob-keller** [comment]
That was also my argument. If you don't participate in specific savage
raids from which the terminology arose it makes no sense and left myself
and a friend confused.

I agree that spread out isn't really correct either though.

File: ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts:99]: I hear you, but I think there's not a better word for "spread out, but in relation to your angle to the boss" vs "spread out, in relation to each other" than "protean".  I think they are different and it's better to differentiate what the mechanic actually is and have a little confusion, and if folks want to call it something different then the they can use the cactbot ui to change it.

**Comment by jacob-keller** [comment]
@quisquous on a related note, during the level 87 dungeon there is an attack on the final boss which is probably mis-labeled as "spread" when its a protean (line attak originating from the boss targeting every player).

This further adds evidence that we're inconsistent. I personally feel that a compromise like "protean spread" or similar would be easier to understand for new players and include the term to differentiate it...

**Comment by quisquous** [comment]
I think ultimately my feeling is that communication is hard, and that sometimes the first time you see a mechanic it might be confusing what cactbot is asking you to do.  I think it's better to use consistent language everywhere and then the next time people see "protean" they'll know what it's asking them to do, and if they don't like it, then they can change it in the cactbot config.

Re: 87 boss, sure we could change that to protean, although it's kind of more of a "protean laser" vs the typical "protean (cone)" since it has a width that a normal protean doesn't have.

**Pull request closed without merging**